public/constants/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export enum ServiceEndpoints {
+  GetIndexes = '/api/relevancy/search/indexes',
+  GetSearchResults = '/api/relevancy/search',
+}
```

Comment:
This `enum` is used by both public layer and server layer. Let's move this to root folder `common/index.ts`where global constants like `PLUGIN_ID` are declared